src/platform_impl/ios/view.rs

```diff
@@ -218,6 +218,8 @@ unsafe fn get_window_class() -> &'static Class {
                         break;
                     }
                     let location: CGPoint = msg_send![touch, locationInView: nil];
+                    let force: CGFloat = msg_send![touch, force];
+                    let max_force: CGFloat = msg_send![touch, maximumPossibleForce];
```

Comment:
This needs a check to be sure force touch is supported on the device/in the os settings. Otherwise `force` is `NaN`.
```rust
let trait_collection: id = msg_send![object, traitCollection];
let touch_capability: UIForceTouchCapability = msg_send![trait_collection, forceTouchCapability];
let force = if touch_capability == UIForceTouchCapability::Available {
    let force: CGFloat = msg_send![touch, force];
    let max_force: CGFloat = msg_send![touch, maximumPossibleForce];
    Some((force / max_force) as _)
} else {
    None
};
```

Reply:
Good catch! I've added the capability check.